astroplan/conftest.py

```diff
@@ -18,17 +18,33 @@
 from .utils import _mock_remote_data, _unmock_remote_data
 from .exceptions import AstroplanWarning
 
+import os
+
+# This is to figure out the affiliated package version, rather than
+# using Astropy's
+try:
+    from .version import version
+except ImportError:
+    version = 'dev'
+
+packagename = os.path.basename(os.path.dirname(__file__))
+TESTED_VERSIONS[packagename] = version
```

Comment:
This is due to unrelated conda issues (yet another one, these past couple of days are quite messed up), simply a too old astropy is picked up. 

So I would wait with this until those things in astropy/ci-helpers#98 are ironed out.